src/java/org/apache/cassandra/net/ResponseVerbHandler.java

```diff
@@ -56,6 +67,56 @@ public void doVerb(Message message)
         {
             MessagingService.instance().latencySubscribers.maybeAdd(cb, message.from(), latencyNanos, NANOSECONDS);
             cb.onResponse(message);
+            trackReplicaSensors(callbackInfo, message);
```

Comment:
CNDB integ tests with 1 or 2 replicas will always return before the sensors tracking finishes . Not sure if the same behavior will manifest in reality. We could track just before `onResponse` to guarantee values (at least from quorum) at the expense of some latency (it is just copying from custom headers to memory, could benchmark as needed).

thoughts?

Reply:
Fixed: https://github.com/datastax/cassandra/pull/1295/commits/f137fa485849a248d8737b705060e7490f9a6805